### PR TITLE
USWDS-Site Pattern pages: Fix broken federalist links found in language selector section.

### LINKED ISSUE #2558: USWDS-Site: Address broken federalist links on language patterns pages.
@finekatie noted a recent drop in USWDS's Siteimprove score. Fixing broken links can improve our score and the experience for our community. 

@amyleadem noted that tools the dev team uses to detect broken links are not currently scanning the [Select a language](https://designsystem.digital.gov/patterns/select-a-language/) pattern section of the site. This pattern section includes several broken Federalist links in the "Pattern preview" section of each subpage. This ticket captures the work of fixing those links.

Including @finekatie on this issue because she is working on a list of broken links to be attached here.

@amyleadem and @mahoneycm, please include a content strategist as a reviewer for this issue, and please do the following:

- Fix the broken links in the language pattern section of the site
- ~Update scanning protocol so that the language pattern section of the site is included in future scans~ _// This issue should be resolved as a part of #2169_

### PR DESCRIPTION
**Pull request by mahoneycm**
# Summary

Updates cloud pages links to match new pattern. This resolves broken links found on language selector previews.

> [!important]
> We need to update changelog dates before merge

## Related issue

Closes #2558

## Preview link

[Two languages →](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-fix-broken-federalist-links/patterns/select-a-language/two-languages/#pattern-preview-2)

[Three or more languages →](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-fix-broken-federalist-links/patterns/select-a-language/three-or-more-languages/#pattern-preview-2)

[Selected multilingual content →](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-fix-broken-federalist-links/patterns/select-a-language/selected-content/#pattern-preview-2)

## Problem statement

There are broken links on the “Select a language” pattern pages. This was caused due to the cloud pages url pattern changing from `app.cloud.gov` to `sites.pages.cloud.gov`.

## Solution

Update links to match new pattern.

## Testing and review

1. Visit each of the “Select a language” pattern pages.
    - [Two languages →](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-fix-broken-federalist-links/patterns/select-a-language/two-languages/#pattern-preview-2)
    - [Three or more languages →](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-fix-broken-federalist-links/patterns/select-a-language/three-or-more-languages/#pattern-preview-2)
    - [Selected multilingual content →](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-fix-broken-federalist-links/patterns/select-a-language/selected-content/#pattern-preview-2)
3. Click the component preview photo.
4. Confirm it loads the storybook preview of the component.
5. Click the “Demo *component name*” button.
6. Confirm it loads the storybook preview of the component.
7. Confirm changelog is accurate and free of grammatical error.

> [!NOTE]
These pages are being ignored by html-proofer at this time. This is being explored further in #2169.
>

### CODE REVIEW

**Review by amyleadem — CHANGES_REQUESTED** [review]
Thanks for jumping on this so quickly @mahoneycm. I added a comment about using `federalist_base` for these links.

**Review by finekatie — APPROVED** [review]
LGTM

File: _includes/patterns/selected-content.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <a class="media__link"
-  href="https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/develop/iframe.html?id=components-language-selector--three-or-more-languages&amp;args=&amp;viewMode=story">
+  href="https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/develop/iframe.html?id=components-language-selector--three-or-more-languages&amp;args=&amp;viewMode=story">
```
[the remark this answers is not in the document]
Comment (mahoneycm) [reply _includes/patterns/selected-content.html]: Oh great catch! Looks like the only spot on the site that wasn't using that variable. Resolved in [fed601c](https://github.com/uswds/uswds-site/pull/2566/commits/fed601ca1f049c828c8b466e1b16b537cff4f363)

**Review by sarah-sch — APPROVED** [review]
Looks good!

**Review by amyleadem — APPROVED** [review]
This looks good! Thanks @mahoneycm.

> [!Note]
> I went ahead and pushed up a change to the changelogs that replaced `affectsContent` with `affectsGuidance` because the `content` tag is typically reserved for the content _inside_ a component or pattern. Let me know if you have any questions!

**Comment by mejiaj** [comment]
@mahoneycm there are more HTMLProofer issues 

```bash
For the Links > External check, the following failures were found:

* At ./_site/patterns/complete-a-complex-form/establish-trust/index.html:2874:

  External link https://uxdesign.cc/build-trust-with-these-ux-guidelines-f3d547bb2014 failed (status code 520)

* At ./_site/patterns/complete-a-complex-form/progress-easily/index.html:2716:

  External link https://uxdesign.cc/trauma-informed-design-research-69b9ba5f8b08 failed (status code 520)
```

[Source →](https://app.circleci.com/pipelines/github/uswds/uswds-site/6485/workflows/a47355c8-6250-4241-aafa-527ffe6fa753/jobs/27479/parallel-runs/0/steps/0-104)

**Pull request merged**